### PR TITLE
Added New Board - Cytron Maker Pi RP2040

### PR DESCRIPTION
**Pull request by waiweng83**
Added the definition for Cytron Maker Pi RP2040.

### CODE REVIEW

File: ports/raspberrypi/boards/cytron_maker_pi_rp2040/board.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 Scott Shawcroft for Adafruit Industries
```
Comment (jepler) [inline ports/raspberrypi/boards/cytron_maker_pi_rp2040/board.c:6]: Please update the copyright information.  It is appropriate to include your name as well as the name of the company for whom the work was done, if applicable.

**Review by jepler — CHANGES_REQUESTED** [review]
Thank you!  I have one requested change for you to make.

File: ports/raspberrypi/boards/cytron_maker_pi_rp2040/board.c (hunk 1 above)
Comment (waiweng83) [reply ports/raspberrypi/boards/cytron_maker_pi_rp2040/board.c]: I'm sorry I missed that out.
It's already updated.
Thanks.

**Comment by waiweng83** [comment]
I'm wondering if there is a way to bundle a library together for this board?
For example I would like to include the neopixel.mpy together with the build.
Is this possible?

**Comment by ladyada** [comment]
yes you can by 'freezing' a library - then it will be automatically included in the firmware
https://github.com/adafruit/circuitpython/blob/main/ports/atmel-samd/boards/adafruit_rotary_trinkey_m0/mpconfigboard.mk#L23

**Comment by waiweng83** [comment]
> yes you can by 'freezing' a library - then it will be automatically included in the firmware
> https://github.com/adafruit/circuitpython/blob/main/ports/atmel-samd/boards/adafruit_rotary_trinkey_m0/mpconfigboard.mk#L23

Thanks. Should I submit another pull request or just modify this one?

**Comment by ladyada** [comment]
keep modifying this one until its exactly how you want it!

**Comment by waiweng83** [comment]
> keep modifying this one until its exactly how you want it!

Alright. Will update here once it's done.

**Comment by waiweng83** [comment]
> yes you can by 'freezing' a library - then it will be automatically included in the firmware
> https://github.com/adafruit/circuitpython/blob/main/ports/atmel-samd/boards/adafruit_rotary_trinkey_m0/mpconfigboard.mk#L23

How about the library which is not in the frozen library list?
I would like to include the SimpleIO as well.

We use that to generate tone for the buzzer.

**Comment by ladyada** [comment]
you can add any library by using git submodule to add it :)

**Review by jepler — APPROVED** [review]
Thank you.  This looks OK to me now.  This can be merged once the automatic Checks have passed.

**Comment by waiweng83** [comment]
> Thank you. This looks OK to me now. This can be merged once the automatic Checks have passed.

Thanks a lot.

**Pull request merged**